### PR TITLE
Fix: JSON Web Signature Browser Clashing

### PR DESCRIPTION
**Pull request by ipbyrne**
This PR is to fix the issue reported here: https://difdn.slack.com/archives/CFNTVGKT2/p1666182975472729

This will allow our package to run in the browser without issues.

### CODE REVIEW

**Comment by BenjaminMoe** [comment]
Looks good to me. Ping review from @OR13

File: packages/json-web-signature/package.json [hunk 1]
```diff
@@ -59,7 +59,6 @@
     "typescript": "^4.1.3"
   },
   "dependencies": {
-    "@transmute/bls12381-key-pair": "^0.7.0-unstable.66",
```
Comment (OR13) [inline packages/json-web-signature/package.json:62]: good

File: packages/json-web-signature/src/JsonWebKey.ts [hunk 2]
```diff
@@ -32,10 +31,6 @@ const getKeyPairForKtyAndCrv = (kty: string, crv: string) => {
       return Secp256k1KeyPair;
     }
 
-    if (crv === 'BLS12381_G2') {
```
Comment (OR13) [inline packages/json-web-signature/src/JsonWebKey.ts:35]: good, we expect this to change eventually when IETF finishes their work.

File: packages/json-web-signature/src/__tests__/generate.test.ts [hunk 3]
```diff
@@ -44,7 +40,7 @@ options.forEach(opt => {
         });
         expect(verified).toBe(true);
       } catch (e) {
-        expect(e.message).toBe('BLS12381_G2 has no registered JWA');
+        expect(e.message).toBe('getKeyPairForType does not support type: Bls12381G2Key2020');
```
Comment (OR13) [inline packages/json-web-signature/src/__tests__/generate.test.ts:43]: good.

**Review by OR13 — APPROVED** [review]

**Pull request merged**